src/sqlfluff/dialects/dialect_databricks.py

```diff
@@ -826,7 +826,12 @@ class AlterTableStatementSegment(sparksql.AlterTableStatementSegment):
                 "DROP",
                 OneOf("COLUMN", "COLUMNS", optional=True),
                 Ref("IfExistsGrammar", optional=True),
-                Bracketed(
+                OneOf(
+                    Bracketed(
```

Comment:
Can you condense this and the below into `OptionallyBracketed(...)`

Reply:
Done.